financial-templates-lib/ExpiringMultiPartyClient.js

```diff
@@ -76,6 +86,18 @@ class ExpiringMultiPartyClient {
       );
   };
 
+  _isExpired = async liquidation => {
+    const currentTime = await this.emp.methods.getCurrentTime().call();
+    return Number(liquidation.liquidationTime) + this.liquidationLiveness <= currentTime;
```

Comment:
I made this equation look exactly like how it is tested in `Liquidatable.sol` just to make sure I didn't have any off-by-1 errors. Before, the equation was `Number(liquidation.liquidationTime) + this.liquidationLiveness > currentTime`

Liquidatable.sol:
```
modifier withdrawable(uint id, address sponsor) {
        LiquidationData storage liquidation = _getLiquidationData(sponsor, id);
        Status state = liquidation.state;

        // Must be disputed or the liquidation has passed expiry.
        require(
            (state > Status.PreDispute) ||
                ((_getLiquidationExpiry(liquidation) <= getCurrentTime()) && (state == Status.PreDispute))
        );
        _;
    }
```